canvas_manage_course/views.py

```diff
@@ -21,8 +20,10 @@
 
 @require_http_methods(['GET'])
 def tool_config(request):
-    url = "%s://%s%s" % (request.scheme, request.get_host(),
-                         reverse('lti_launch', exclude_resource_link_id=True))
+
+    host = 'https://' + request.get_host()
+    url = host + reverse('lti_launch')
```

Comment:
This works, although using string formatting reads better to me because I can more easily visualize the resulting string.   That said, it's equivalent to the concats in place so no pressure to change:
`url = "https://{0}{1}".format(request.get_host(), reverse('lti_launch'))`